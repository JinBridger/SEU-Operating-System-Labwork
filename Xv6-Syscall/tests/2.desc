concurrency test; did you protect the counter with a lock?
