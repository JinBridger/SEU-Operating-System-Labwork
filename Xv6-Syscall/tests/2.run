cd src; ../../tester/run-xv6-command.exp CPUS=2 Makefile.test test_2 | grep XV6_TEST_OUTPUT; cd ..
