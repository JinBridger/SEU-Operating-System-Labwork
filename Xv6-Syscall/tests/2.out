XV6_TEST_OUTPUT 200000
